UVAtlasTool/Mesh.h

```diff
@@ -143,6 +146,9 @@ class Mesh
         }
     };
 
+    void ExportToOBJ(const wchar_t* szFileName) const;
+    void ExportToOBJ(std::string filePath) const;
```

Comment:
Personally, I'd rather just see one method:

```
HRESULT ExportToOBJ(const wchar_t* szFileName) const;
```

That said if you feel strongly about the fstream version, you should move the ``#include <fstream>`` to the Mesh.h since it's part of the public interface.:

```
HRESULT ExportToOBJ(std::ostream& os) const;
```

Reply:
Made it private instead. Don't want to delete it now that it's coded. It will be useful for whomever tries to extend this ExportToOBJ later